tests/testthat/in/parsable-roxygen-fail2.R

```diff
@@ -0,0 +1,18 @@
+#' Some function
+#' 
+#' This function is great! But \code{oh dear, a missing brace...
+#' 
+#' And isn't that a multi-line example? We should probably use @examples...
+#' 
+#' @param x A parameter.
+#'
+#' @returns Invisible `NULL`.
+#' 
+#' @example
+#' some_function(10)
+#' some_function(11)
+#' 
+#' @export
+some_function <- function(x) {
```

Comment:
Can you make the roxygen comment more minimal here since you want to test is parsing R code?

Reply:
Done in 1bf87ac